astropy/vo/client/tests/test_conesearch.py

```diff
@@ -149,6 +149,9 @@ def test_async(self):
         async_search = conesearch.AsyncConeSearch(
             SCS_CENTER, SCS_RADIUS, pedantic=self.pedantic)
 
+        # Wait a little for the instance to set up properly
+        time.sleep(3)
```

Comment:
@MSeifert04 , I do see your concern about forcing the test to wait, which increases total test time. If that is an issue, we can simply mark these two `async` tests as `xfail`.